tests/jeos/rpi_wifi.pm

```diff
@@ -0,0 +1,39 @@
+# SUSE's openQA tests
+#
+# Copyright 2015-2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Check RPi Wifi
+# Maintainer: qsf-u team <qe-core@suse.de>
+
+use base "opensusebasetest";
+use strict;
+use warnings;
+use testapi;
+use Utils::Architectures;
+use Utils::Backends;
+
+sub run {
+    my ($self) = @_;
+
+    assert_script_run 'ip l';
+    assert_script_run 'rfkill list';
+    assert_script_run 'ip l set wlan0 up';
+    # This would be a more modern approach but
+    # iw is not installed by default:
+    # iw dev wlan0 scan | grep -e "^\(BSS\|\sSSID:\)"
+    assert_script_run 'iwlist wlan0 scan | grep -e "^\s*\(Cell\|Frequency\|Quality\|ESSID\)"';
+    assert_script_run 'ip l set wlan0 down';
+
+    #TODO: Actually connect to openQA-worker wifi
```

Comment:
Don't add TODOs as likely this will stay here forever. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit or put it into tickets which are more suitable for tracking tasks

See https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md#coding-style for more details